client/my-sites/site/style.scss

```diff
@@ -157,3 +156,72 @@
 .site__content:hover .site__home {
 	opacity: 1;
 }
+
+.site__toggle-more-options {
+	color: $gray;
+	cursor: pointer;
+	padding-right: 16px;
+	line-height: 0;
+	.gridicon {
+		transition: transform 0.15s cubic-bezier(0.175, 0.885, 0.32, 1.275);
+	}
+	&:hover {
+		.gridicon {
+			color: darken( $gray, 20% );
+		}
+	}
+}
+
+.site.is-toggled {
+	.site__toggle-more-options .gridicon {
+		transform: rotate( 90deg );
+	}
+	.site-icon {
+		border: 1px dashed $gray-dark;
```

Comment:
This doesn't account for users who don't have the capability for editing a site's option, in which case it still shows the dashed border, which is a bit misleading:

![image](https://cloud.githubusercontent.com/assets/1779930/12795917/5a67c9b0-ca8a-11e5-91af-26295445643b.png)